data/ms/legislature/Kevin-Horan-b339d980-af38-4e76-b53c-215c26191cc1.yml

```diff
@@ -5,7 +5,9 @@ family_name: Horan
 email: khoran@house.ms.gov
 image: http://billstatus.ls.state.ms.us/members/house/horan.jpg
 party:
-- name: Democratic
+- end_date: '2022-12-29'
```

Comment:
@NewAgeAirbender This person's end date should be january 2, 2024. In 2020 he changed from Democrat to Republican so their party should be republican